ajna/migrations/0102_v4hemiactivitysnapshot_v4hemiauction_and_more.py

```
@@ -0,0 +1,575 @@
+                ('block_datetime', models.DateTimeField()),
+            ],
+            options={
+                'abstract': False,
+                'indexes': [models.Index(fields=['pool_address', 'bucket_index', 'block_number'], name='ajna_v4hemi_pool_ad_f66e1a_idx'), models.Index(fields=['pool_address', 'bucket_index'], name='ajna_v4hemi_pool_ad_92e29d_idx'), models.Index(fields=['pool_address'], name='ajna_v4hemi_pool_ad_c3027e_idx'), models.Index(fields=['bucket_index'], name='ajna_v4hemi_bucket__e6beb1_idx'), models.Index(fields=['block_number'], name='ajna_v4hemi_block_n_e99d39_idx'), models.Index(fields=['block_datetime'], name='ajna_v4hemi_block_d_0cbdf4_idx')],
+                'unique_together': {('pool_address', 'bucket_index', 'block_number')},
+            },
+        ),
+        migrations.CreateModel(
+            name='V4HemiPoolEvent',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
```

Ruff (0.11.9)

Line too long (117 > 100)

(E501)